lib/ClassNameInflector.php

```diff
@@ -36,10 +36,27 @@ class ClassNameInflector
      */
     public static function getUserClassName($className)
     {
+        return self::getInflector()->getUserClassName($className);
+    }
+
+    /**
+     * Return true if the given class name appears to be the name of a proxy class.
+     *
+     * @param string $className
+     *
+     * @return bool
+     */
+    public static function isProxyClassName($className)
```

Comment:
Are you assuming, or is it really needed? It was obviously not needed until now, so have you added calls to this method?